wp-api.js

```diff
@@ -10,8 +10,8 @@
 	};
 
 	/**
-	 * ECMAScript 5 shim, from MDN
-	 * https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString
+	 * ECMAScript 5 shim, from MDN.
+	 * https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString.
```

Comment:
I'm not sure it makes sense to add a period here, as it changes the nature of the URL.